src/Compilers/CSharp/Test/Symbol/Symbols/ExtendedPartialMethodsTests.cs

```
@@ -0,0 +1,552 @@
+            );
+        }
+
+        [Fact]
+        public void Virtual_Semantics()
+        {
+            const string text1 = @"
+using System;
+
+public partial class C
+{
+    internal virtual partial int M1();
+    internal virtual partial int M1() => 1;
+}
+
+public class D : C
+{
+    internal override int M1() => 2;
+
+    static void UseC(C c)
+    {
+        Console.Write(c.M1());
+    }
+
+    public static void Main()
+    {
+        UseC(new C());
+        UseC(new D());
+    }
+}";
+            CompileAndVerify(text1, parseOptions: TestOptions.RegularWithExtendedPartialMethods, expectedOutput: "12").VerifyDiagnostics();
+        }
+
+        [Fact]
+        public void Override_LangVersion()
+        {
+            const string text1 = @"
+partial class D
+{
+    internal override partial string ToString();
+    internal override partial string ToString() => ""hello"";
+}";
+            var comp = CreateCompilation(text1);
+            comp.VerifyDiagnostics(
+                // (4,38): error CS8652: The feature 'extended partial methods' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                //     internal override partial string ToString();
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "ToString").WithArguments("extended partial methods").WithLocation(4, 38),
+                // (5,38): error CS8652: The feature 'extended partial methods' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                //     internal override partial string ToString() => "hello";
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "ToString").WithArguments("extended partial methods").WithLocation(5, 38)
+            );
+            comp = CreateCompilation(text1, parseOptions: TestOptions.RegularWithExtendedPartialMethods);
+            comp.VerifyDiagnostics();
```

>VerifyDiagnostics [](start = 17, length = 17)

Shouldn't we report an error that the `override` of `ToString()` is `internal`?

---

Weird, I will look into why this is happening.

---

Fixed